src/targets/happymodel_900.ini

```diff
@@ -50,3 +63,9 @@ upload_flags =
 [env:HappyModel_RX_ES915RX_via_BetaflightPassthrough]
 extends = env:HappyModel_RX_ES915RX_via_STLINK
 
+[env:HappyModel_RX_ES900RX_via_WIFI]
```

Comment:
No `HappyModel_RX_ES900RX_via_STLINK` target?

Reply:
Nope. These are ESP8285 based targets on the RX, and ESP32 on TX